cxx4opencl/address_spaces.txt

```diff
@@ -185,19 +260,24 @@ __kernel void bar() {
   c1.foo(); // will resolve to the first 'foo'.
   c2.foo(); // will resolve to the second 'foo'.
   c3.foo(); // error due to mismatching address spaces - can't convert to
-            // '__local' or generic addr spaces.
+            // '__local' or generic/'__private' addr spaces.
 }
-----------
+------------
+
+All  member functions can be qualified by an address space qualifier including
+constructors and destructors.
```

Comment:
It might be good for explicitness to have an example with a constructor. (There's an example under Construction, initialization and destruction but it's much later in this document.)